src/System.Runtime/tests/System/Reflection/AssemblyTests.cs

```diff
@@ -197,8 +197,16 @@ public static void GetModules_GetModule(Assembly assembly)
             Assert.NotEmpty(assembly.GetModules());
             foreach (Module module in assembly.GetModules())
             {
-                Assert.NotNull(module);
-                Assert.Equal(module, assembly.GetModule(module.Name));
```

Comment:
Hmm, I get the following test failures: `msbuild /T:BuildAndTest /P:TargetGroup=netfx`

```
      System.Reflection.Tests.AssemblyTests.GetModules_GetModule(assembly: mscorlib, Version=4.0.0.0, Culture=neutral,
  PublicKeyToken=b77a5c561934e089) [FAIL]
        Assert.Equal() Failure
        Expected: CommonLanguageRuntimeLibrary
        Actual:   (null)
        Stack Trace:
          C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime\tests\System\Reflection\AssemblyTests.cs(200,0): at
   System.Reflection.Tests.AssemblyTests.GetModules_GetModule(Assembly assembly)
      System.Reflection.Tests.AssemblyTests.GetModules_GetModule(assembly: mscorlib, Version=4.0.0.0, Culture=neutral,
  PublicKeyToken=b77a5c561934e089) [FAIL]
        Assert.Equal() Failure
        Expected: CommonLanguageRuntimeLibrary
        Actual:   (null)
        Stack Trace:
          C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime\tests\System\Reflection\AssemblyTests.cs(200,0): at
   System.Reflection.Tests.AssemblyTests.GetModules_GetModule(Assembly assembly)
```